### PR TITLE
feat(TDI-45016):add query time out.

### PR DESCRIPTION
**Pull request by vyu-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-45016

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tAS400Output/tAS400Output_begin.javajet [hunk 1]
```diff
@@ -174,43 +178,47 @@ if(columnList != null && columnList.size() > 0) {
     StringBuilder updateSetStmt = actionSQLMap.get(UPDATE_SET_STMT);
     StringBuilder updateWhereStmt = actionSQLMap.get(UPDATE_WHERE_STMT);
     StringBuilder deleteWhereStmt = actionSQLMap.get(DELETE_WHERE_STMT);
-
+    %>
+    java.sql.PreparedStatement pstmt_<%=cid %> = null;
+    java.sql.PreparedStatement pstmtInsert_<%=cid %> = null;
+    java.sql.PreparedStatement pstmtUpdate_<%=cid %> = null;
```
Comment (pyzhou-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tAS400Output/tAS400Output_begin.javajet:184]: why move these declarations out?

**Review by pyzhou-talend — APPROVED** [review]
LGTM

File: main/plugins/org.talend.designer.components.localprovider/components/tAS400Output/tAS400Output_begin.javajet (hunk 1 above)
Comment (pyzhou-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tAS400Output/tAS400Output_begin.javajet:184]: I know it works the same way. but I didn't see the need of moving them out.

Comment (vyu-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tAS400Output/tAS400Output_begin.javajet:184]: In order to make the code more concise, I optimized code.

**Review by pyzhou-talend — APPROVED** [review]
LGTM

**Pull request merged**